xe/thinCLIProtocol.cs

```diff
@@ -543,7 +540,7 @@ private static void Interpreter(Stream stream, ThinCliProtocol tCLIprotocol)
             string path;
             string msg;
 
-            while (!tCLIprotocol.dropOut)
+            while (true)
```

Comment:
Isnt this a continuous loop?

Reply:
Yes, it is broken when we receive an Exit.Tag from the server (line 601) or a ThinProtocolError is thrown (previously this was `while (!tCLIprotocol.dropOut)`, but dropOut was initialised with false and was never set).